servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -314,64 +303,66 @@ def _collect_change_requests(self, instance_info):
+                    change_request.number.display_value,
+                    change_request.custom_cmdb_ci.value,
+                    change_request.custom_cmdb_ci.display_value,
+                    change_request.sys_updated_on.value,
+                    change_request.sys_updated_on.display_value
                 )
-                if change_request.sys_updated_on > instance_info.state.latest_sys_updated_on:
-                    instance_info.state.latest_sys_updated_on = change_request.sys_updated_on
-                old_state = instance_info.state.change_requests.get(change_request.number)
-                if old_state is None or old_state != change_request.state:
+                if change_request.sys_updated_on.value > state.latest_sys_updated_on:
+                    state.latest_sys_updated_on = change_request.sys_updated_on.value
+                old_state = state.change_requests.get(change_request.number.display_value)
+                if old_state is None or old_state != change_request.state.display_value:
                     self._create_event_from_change_request(change_request)
-                    instance_info.state.change_requests[change_request.number] = change_request.state
+                    state.change_requests[change_request.number.display_value] = change_request.state.display_value
 
     def _create_event_from_change_request(self, change_request):
-        host = Identifiers.create_host_identifier(to_string(change_request.cmdb_ci['display_value']))
-        identifiers = [change_request.cmdb_ci['value'], host]
-        timestamp = (change_request.sys_updated_on - datetime.datetime.utcfromtimestamp(0)).total_seconds()
-        msg_title = '%s: %s' % (change_request.number, change_request.short_description)
+        host = Identifiers.create_host_identifier(to_string(change_request.custom_cmdb_ci.display_value))
+        identifiers = [change_request.custom_cmdb_ci.value, host]
+        timestamp = (change_request.sys_updated_on.value - datetime.datetime.utcfromtimestamp(0)).total_seconds()
+        msg_title = '%s: %s' % (change_request.number.display_value, change_request.short_description.display_value)
         tags = [
-            'number:%s' % change_request.number,
-            'priority:%s' % change_request.priority,
-            'risk:%s' % change_request.risk,
-            'state:%s' % change_request.state,
-            'category:%s' % change_request.category,
-            'conflict_status:%s' % change_request.conflict_status,
-            'assigned_to:%s' % change_request.assigned_to
+            'number:%s' % change_request.number.display_value,
+            'priority:%s' % change_request.priority.display_value,
+            'risk:%s' % change_request.risk.display_value,
+            'state:%s' % change_request.state.display_value,
+            'category:%s' % change_request.category.display_value,
+            'conflict_status:%s' % change_request.conflict_status.display_value,
```

Comment:
This used to be `value` , should it be `display_value` instead?

Reply:
That was a typo